src/app/static/src/tests/components/runReport/parameterList.test.ts

```diff
@@ -0,0 +1,43 @@
+import {shallowMount} from "@vue/test-utils";
+import ParameterList from "../../../js/components/runReport/parameterList.vue"
+import Vue from "vue"
+
+
+describe(`run report parameter list`, () => {
+    const params = [
+        {name: "global", default: ""},
+        {name: "minimal", default: ""}
+    ]
+
+    const store = () => shallowMount(ParameterList,
+        {
+            propsData: {params: params}
+        }
+    )
+
+    it(`can render parameter labels and values as expected`, () => {
+        const wrapper = store()
+        const labels = wrapper.find("table").findAll("label")
+
+        expect(labels.at(0).text()).toBe("global")
+        expect(labels.at(1).text()).toBe("minimal")
+        expect(wrapper.vm.$data.paramValues).toBe(params)
```

Comment:
Would be good to also check that the form inputs are populated with default parameters values provided by server

Reply:
This test case was never testing inputs. perhaps the test name was confusing, I have renamed it. Below this test case is the test for input.